### PR TITLE
button group a11y compliance

### PR DESCRIPTION
**Pull request by xman343**
## Changes

Bringing `ButtonGroup` examples to a11y compliance. `ButtonGroup` component brought to compliance by removing `aria-orientation` attribute .

## Testing

Unit testing & visual testing will be conducted. 

## Docs

TBD

### CODE REVIEW

File: examples/ButtonGroup.main.tsx [hunk 1]
```diff
@@ -14,16 +14,16 @@ import {
 export default () => {
   return (
     <ButtonGroup>
-      <IconButton onClick={() => {}}>
+      <IconButton onClick={() => {}} label='add'>
```
Comment (mayank99) [inline examples/ButtonGroup.main.tsx:17]: can you change all these labels (and aria-labels) to be **Sentence case**?

File: .changeset/ten-grapes-approve.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': patch
+---
+
+Orientation attribute removed from ButtonGroup to meet a11y accessibility requirements.
```
Comment (mayank99) [inline .changeset/ten-grapes-approve.md:5]:
```suggestion
`aria-orientation` attribute removed from ButtonGroup to meet accessibility requirements.
```

File: examples/ButtonGroup.main.tsx (hunk 1 above)
Comment (xman343) [reply examples/ButtonGroup.main.tsx]: Done!

File: examples/ButtonGroup.input.tsx [hunk 3]
```diff
@@ -17,13 +17,16 @@ export default () => {
...
-        <IconButton>
+        <Input aria-label='search bar' />
+        <IconButton label='search'>
           <SvgSearch />
         </IconButton>
       </ButtonGroup>
       <ButtonGroup>
-        <Input value='https://itwinui.bentley.com/docs/buttongroup' />
+        <Input
+          aria-label='input text bar'
```
[the remark this answers is not in the document]
Comment (xman343) [reply examples/ButtonGroup.input.tsx]: Done

File: .changeset/ten-grapes-approve.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': patch
+---
+
+`aria-orientation` attribute removed from ButtonGroup to meet accessibility requirements.
```
Comment (adamhock) [inline .changeset/ten-grapes-approve.md:5]: I would mention this in the PR description

**Review by adamhock — APPROVED** [review]

File: .changeset/ten-grapes-approve.md (hunk 4 above)
Comment (xman343) [reply .changeset/ten-grapes-approve.md:5]: Done!

**Pull request merged**